README.md

```diff
@@ -4,21 +4,402 @@ Complete the following queries and paste the sql results under each question. Us
 
 
 1. Select all the customers whose first name starts with “L” and is from Germany
+
+SELECT * FROM Customers
+WHERE CustomerName LIKE "L%"
+AND Country="Germany";
+
+Number of Records: 1
+
+CustomerID	CustomerName	ContactName	Address	City	PostalCode	Country
+44	Lehmanns Marktstand Renate Messner	Magazinweg 7	Frankfurt a.M.	60528	Germany
+
+
 2. Select all the product names that are in the “Condiments” category
+
+SELECT ProductName, CategoryName
+FROM Products
+INNER JOIN Categories
+ON Products.CategoryID=Categories.CategoryID
+WHERE CategoryName="Condiments";
+
+Number of Records: 12
+
+ProductName	CategoryName
+Aniseed Syrup	Condiments
+Chef Anton's Cajun Seasoning	Condiments
+Chef Anton's Gumbo Mix	Condiments
+Grandma's Boysenberry Spread	Condiments
+Northwoods Cranberry Sauce	Condiments
+Genen Shouyu	Condiments
+Gula Malacca	Condiments
+Sirop d'érable	Condiments
+Vegie-spread	Condiments
+Louisiana Fiery Hot Pepper Sauce	Condiments
+Louisiana Hot Spiced Okra	Condiments
+Original Frankfurter grüne Soße	Condiments
+
 3. Select the customer names who have never had an order
+
+SELECT CustomerName, OrderID
+FROM Customers
+LEFT JOIN Orders
+ON Customers.CustomerID=Orders.CustomerID
+WHERE Orders.CustomerID IS NULL
+
+Number of Records: 17
+CustomerName	OrderID
+Alfreds Futterkiste	null
+Blauer See Delikatessen	null
+Cactus Comidas para llevar	null
+FISSA Fabrica Inter. Salchichas S.A.	null
+France restauration	null
+Great Lakes Food Market	null
+La corne d'abondance	null
+Laughing Bacchus Wine Cellars	null
+Lazy K Kountry Store	null
+Let's Stop N Shop	null
+Maison Dewey	null
+North/South	null
+Paris spécialités	null
+Rancho grande	null
+Spécialités du monde	null
+The Cracker Box	null
+Trail's Head Gourmet Provisioners	null
+
 4. Select the list of customer_ids, and the number or orders, sorted descending
+
+SELECT Customers.CustomerID, COUNT(Orders.OrderID) as [Total Count] FROM Customers
+LEFT JOIN Orders
+ON Orders.CustomerID = Customers.CustomerID
+Group By Customers.CustomerID
+ORDER BY [Total Count] Desc;
+
+Number of Records: 91
+CustomerID	Total Count
+20	10
+63	7
+65	7
+87	7
+37	6
+75	6
+41	5
+46	5
+51	5
+7	4
+10	4
+24	4
+25	4
+55	4
+61	4
+71	4
+80	4
+86	4
+5	3
+9	3
+21	3
+36	3
+38	3
+44	3
+49	3
+59	3
+60	3
+66	3
+69	3
+72	3
+4	2
+14	2
+17	2
+19	2
+28	2
+29	2
+34	2
+35	2
+39	2
+48	2
+58	2
+62	2
+67	2
+68	2
+73	2
+76	2
+81	2
+83	2
+84	2
+85	2
+88	2
+89	2
+2	1
+3	1
+8	1
+11	1
+13	1
+15	1
+16	1
+18	1
+23	1
+27	1
+30	1
+31	1
+33	1
+47	1
+52	1
+54	1
+56	1
+70	1
+77	1
+79	1
+90	1
+91	1
+1	0
+6	0
+12	0
+22	0
+26	0
+32	0
+40	0
+42	0
+43	0
+45	0
+50	0
+53	0
+57	0
+64	0
+74	0
+78	0
+82	0
+
 5. Select the SUM of all orders in 1997
+
+SELECT Count(OrderID) FROM [Orders]
+where OrderDate like '1997-%'
```

Comment:
This works but is not the ideal scenario. Like you mentioned the date functions aren't totally working...

How would you do this using ```EXTRACT()```? http://www.w3schools.com/sql/func_extract.asp